2021-06-09 01:13:46,144 DEBUG    root            Loaded Command Group: ['gcloud', 'functions']
2021-06-09 01:13:46,164 DEBUG    root            Loaded Command Group: ['gcloud', 'functions', 'deploy']
2021-06-09 01:13:46,181 DEBUG    root            Running [gcloud.functions.deploy] with arguments: [--allow-unauthenticated: "True", --trigger-http: "True", NAME: "timeleft-develop"]
2021-06-09 01:13:46,197 DEBUG    urllib3.connectionpool Starting new HTTPS connection (1): cloudfunctions.googleapis.com:443
2021-06-09 01:13:47,535 DEBUG    urllib3.connectionpool https://cloudfunctions.googleapis.com:443 "GET /v1/projects/timeleft-develop/locations/us-central1/functions/timeleft-develop?alt=json HTTP/1.1" 404 None
2021-06-09 01:13:47,537 DEBUG    root            (gcloud.functions.deploy) Missing required argument [runtime]: Flag `--runtime` is required for new functions.
Traceback (most recent call last):
  File "/Users/admin/Downloads/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 982, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/Users/admin/Downloads/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 809, in Run
    resources = command_instance.Run(args)
  File "/Users/admin/Downloads/google-cloud-sdk/lib/surface/functions/deploy.py", line 78, in Run
    return command_v1.Run(args, track=self.ReleaseTrack())
  File "/Users/admin/Downloads/google-cloud-sdk/lib/googlecloudsdk/command_lib/functions/v1/deploy/command.py", line 264, in Run
    raise calliope_exceptions.RequiredArgumentException(
googlecloudsdk.calliope.exceptions.RequiredArgumentException: Missing required argument [runtime]: Flag `--runtime` is required for new functions.
2021-06-09 01:13:47,539 ERROR    root            (gcloud.functions.deploy) Missing required argument [runtime]: Flag `--runtime` is required for new functions.
